plugin/pkg/admission/runtimeclass/admission.go

```diff
@@ -193,6 +231,28 @@ func validateOverhead(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta
 	return nil
 }
 
+func validateScheduling(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.RuntimeClass) (err error) {
```

Comment:
I'm not sure this method is needed. Tolerations are only appendable, and the nodeselector is immutable.

Reply:
It basically does the same thing as `setScheduling` which validates the node selector between pod and scheduling.

Reply:
Right, I suppose it's possible another admission controller made changes.